tsdb/fileutil/fileutil.go

```diff
@@ -59,13 +59,20 @@ func CopyDirs(src, dest string) error {
 	return nil
 }
 
+// Making changes for running benchmark in Prometheus.
```

Comment:
This comment is unnecessary, we would want that in case the benchmarks look good and the PR gets merged.